woocommerce/woocommerce-android
# Creating shipping label: Shipment details bottom sheet & Address card

Part of: #12281

### Description
The changes in this PR include the shipment details bottom sheet and the Address card implementation. All the changes in this PR are focused on UI, and the data used in the screens is mocked. On a different PR, I will connect this UI with real data.

Considering that the Address card will be worked on at a different milestone, the current implementation does not include the Ship from selection or the edit Ship to address. 

I didn't work on the shipment details title animation in this PR. I'm planning some time to work on this animation after the connection with real data

### Testing information

1. Create an Order with the processing status on a store that can create shipping labels
2. Tap on Orders
3. Tap on the order created on step 1
4. Tap con create shipping labels
5. Check that Shipment details card is displayed at the bottom of the screen
6. Tap on/Drag up Shipment details
7. Check that the bottom sheets expand and the Address card is visible
8. Tap on/Drag down the bottom sheet handle
9. Check that the bottom sheet collapses

### The tests that have been performed
1. Test that the UI interaction works as expected

### Images/gif

https://github.com/user-attachments/assets/7037cbeb-0c58-44bb-9426-2fc5fed31b5e


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
